src/os/OSSelectWakeup.hxx

```diff
@@ -180,7 +180,7 @@ public:
             ret = -1;
             errno = EINTR;
         }
-#elif defined(__WINNT__) || defined(ESP_NONOS)
+#elif defined(__WINNT__) || defined(ESP_NONOS) || defined(ESP32)
```

Comment:
currently not, by going with __FreeRTOS__ it may work but it will have a lot more changes and issues with it. I'll do a test with that if I can before Friday.